server/models/models.go

```diff
@@ -36,8 +36,11 @@ type Evolution struct {
 // Account contains the information for each user
 type Account struct {
 	// Username and Email are unique Identifiers
-	Username, Password, Email, Salt string
-	ProjectIds                      []string
+	Username string
+	Password string 	`json:"-"`
```

Comment:
this will prevent us from receiving the Password as well. I think right now it might be best to use an anonymous struct when we write the response that just has `Username` and `Password` since it's a small amount of output.

Reply:
notice the failed test aswell, added a new struct first hand but a anonymous struct sounds good, adding a new struct just for this case seems "to much" - thanks!